src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstLeg * secondLeg) / 2;
+    }
+
+    @Override
+    protected String getName() {
+        return "right triangle";
+    }
+
+    @Override
+    protected String getProperties() {
+        return "firstLeg: " + firstLeg + " units, secondLeg: " + secondLeg + " units";
+    }
+}

```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the start of the class definition (line 2).